src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -1646,4 +1650,35 @@ public long countGermplasmByStudy(final Integer studyDbId) {
 			throw new MiddlewareQueryException(message, e);
 		}
 	}
+
+	public Map<Integer, StudyGermplasmDto> getPlotNoToStudyGermplasmDtoMap(final Integer studyId, final Set<Integer> plotNos) {
```

Comment:
@darla-leafnode reviewing the code for the demo tomorrow, I dont get why we need to map this inside the DAO? can not this be done at the service level if getting the StudyGermplasmDto by studyId is useful for another feature?

Reply:
Yes, actually the map can be actually be formed at the service-level.
@mderamos